ts/Core/Axis/Axis3DComposition.ts

```diff
@@ -447,7 +449,7 @@ class Axis3DAdditions {
         Tick3D.compose(TickClass);
 
         if (U.pushUnique(composedMembers, AxisClass)) {
-            merge(true, AxisClass.defaultOptions, Axis3DDefaults);
+            merge(true, defaultOptions.xAxis, Axis3DDefaults);
```

Comment:
This might be replaceable with `D.setOptions`.

Reply:
I think this one is simpler because it goes directly to the xAxis sub-object.